sandbox/rocky/tf/misc/tensor_utils.py

```diff
@@ -118,3 +118,21 @@ def pad_tensor_dict(tensor_dict, max_len):
         else:
             ret[k] = pad_tensor(tensor_dict[k], max_len)
     return ret
+
+class sys_op_scope(object):
```

Comment:
`sys_op_scope()` is pretty generic and might be confused with a native TensorFlow feature. Maybe it we can call it `enclosing_scope()`.